src/rooms/RestartRoom.ts

```diff
@@ -16,7 +16,12 @@ export class RestartRoom extends Room<RestartRoomState> {
     console.log("Room created with ID: ", this.roomId);
     // Create logic to dispose rooms after a certain time if there is <= 1 connected client
     this.autoDispose = false;
-    // make all these rooms private for added security
+    this.clock.setInterval(() => {
```

Comment:
This will fail. Case: T0 - 0 players, T10 - 1 player, T30 - 5 players, T59 - 1 player, T60 -> disconnect. This should not happen. i.e. When the second player joins, the Interval created on onCreate should be deleted.